production/README.md

```diff
@@ -218,86 +222,86 @@ addnode=liquidtestnet.com:18891
 addnode=liquid.network:18444
 ```
 
-Start elementsd and wait for it to sync the Liquid blockchain.
+Start `elementsd` and wait for it to sync the Liquid blockchain.
 
 ### Electrs
 
 Install [Electrs](https://github.com/Blockstream/electrs) from source:
 ```
-git clone https://github.com/Blockstream/electrs
-cd electrs
-git checkout new-index
+$ git clone https://github.com/Blockstream/electrs
+$ cd electrs
+$ git checkout new-index
 ```
 
 You'll need one instance per network. Build and run them one at a time:
 ```
-./electrs-start-mainnet
-./electrs-start-testnet
-./electrs-start-signet
-./electrs-start-liquid
-./electrs-start-liquidtestnet
+$ ./electrs-start-mainnet
+$ ./electrs-start-testnet
+$ ./electrs-start-signet
+$ ./electrs-start-liquid
+$ ./electrs-start-liquidtestnet
 ```
 
 ### MariaDB
 
-Import historical mempool fee database snapshot:
+Import the historical mempool fee database snapshot:
 ```
-mysql -u root
-create database mempool;
-grant all on mempool.* to 'mempool'@'localhost' identified by 'mempool';
-create database mempool_testnet;
-grant all on mempool_testnet.* to 'mempool_testnet'@'localhost' identified by 'mempool_testnet';
-create database mempool_signet;
-grant all on mempool_signet.* to 'mempool_signet'@'localhost' identified by 'mempool_signet';
-create database mempool_liquid;
-grant all on mempool_liquid.* to 'mempool_liquid'@'localhost' identified by 'mempool_liquid';
-create database mempool_liquidtestnet;
-grant all on mempool_liquidtestnet.* to 'mempool_liquidtestnet'@'localhost' identified by 'mempool_liquidtestnet';
+$ mysql -u root
+mysql> create database mempool;
```

Comment:
Please don't put dollar signs or mysql prompt in front of these commands, since it breaks the ability to copy and paste the entire block of text now 😅